web-client/integration-tests/journey/docketClerkVerifiesConsolidatesCases.js

```diff
@@ -0,0 +1,10 @@
+export const docketClerkVerifiesConsolidatesCases = cerebralTest => {
+  return it('Docket clerk verifies consolidates cases', () => {
+    expect(cerebralTest.getState('caseDetail')).toHaveProperty(
```

Comment:
Would it be worth it to add a step in this test to route to case detail to ensure we are on the case we expect? Or adding an expectation for the expected docket number? Just a thought

Reply:
Thank you for the suggestion! I added a change here to check for that. However one difference in the implementation is that instead of checking that we are on the expected docketNumber I instead check that the case is part of the same consolidated group by checking the lead docket number associated with that case and comparing it to the lead docket number created at the beginning of the test.

Reply:
I think that is appropriate because the docket number can be any case in the group, as long as it is in the same consolidated case group. If I am wrong or misunderstood please let me know!